From 7126209887b591a5de4ff62bfeca537ebd2b4f0b Mon Sep 17 00:00:00 2001
From: Ankit Agrawal <ankit.agarwal@intel.com>
Date: Thu, 18 May 2023 16:10:30 +0530
Subject: [PATCH] Fixed Build Error in Kernel.

Build Error-:
ld.lld: error: version script assignment of 'LINUX_2.6' to symbol
'__vdso_sgx_enter_enclave' failed: symbol not defined.

Enabling X86_SGX config

Tracked-On: OAM-106853
Signed-off-by: Ankit Agrawal <ankit.agarwal@intel.com>
---
 .../kernel/gmin64/config-lts/lts2021-chromium/x86_64_defconfig  | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/groups/kernel/gmin64/config-lts/lts2021-chromium/x86_64_defconfig b/groups/kernel/gmin64/config-lts/lts2021-chromium/x86_64_defconfig
index 95bf06c..0d6c6dd 100644
--- a/groups/kernel/gmin64/config-lts/lts2021-chromium/x86_64_defconfig
+++ b/groups/kernel/gmin64/config-lts/lts2021-chromium/x86_64_defconfig
@@ -427,7 +427,7 @@ CONFIG_X86_INTEL_MEMORY_PROTECTION_KEYS=y
 CONFIG_X86_INTEL_TSX_MODE_OFF=y
 # CONFIG_X86_INTEL_TSX_MODE_ON is not set
 # CONFIG_X86_INTEL_TSX_MODE_AUTO is not set
-# CONFIG_X86_SGX is not set
+CONFIG_X86_SGX=y
 CONFIG_EFI=y
 # CONFIG_EFI_STUB is not set
 # CONFIG_HZ_100 is not set
-- 
2.17.1

